frontend/src/utils/langqc.js

```diff
@@ -0,0 +1,70 @@
+import { join, reject } from "lodash";
+
+export default class LangQc {
+  constructor(host) {
+    if (typeof host == 'undefined') {
+      throw Error('LangQc client must know where the web service is');
+    }
+    this.host = host;
+
+    this.urls = {
+      inbox: this.buildUrl('pacbio/inbox', ['weeks=1']),
+      run: this.buildUrl('pacbio/run')
+    };
+    this.commonHeaders = {
+      'Accept': 'application/json',
+      'Origin': 'https://*.dnapipelines.sanger.ac.uk',
+      'access-control-allow-origin': 'https://*.dnapipelines.sanger.ac.uk'
```

Comment:
Would be nice to have this configurable (same as API host). Though these headers might not even be needed in the setup with both backend and frontend behind the same reverse-proxy.